### ISSUE TITLE
WriteVTK is only correct for first order cells

### ISSUE THREAD

**Issue by mohamed82008** [KristofferC/JuAFEM.jl#192]
https://github.com/KristofferC/JuAFEM.jl/blob/a8d976d4e9d4bba415c3d2dbf062a75f8d814b2c/src/Dofs/ConstraintHandler.jl#L211

The function `faces` only returns tuples of corner points, so the boundary conditions on non-corner points will not be written.

**Comment by fredrikekre** [comment]
Since currently only put BCs on faces I think this was just the simplest way to color a face.

**Comment by mohamed82008** [comment]
Yes, I get the point of the function. I drafted some node bc functionality but currently debugging it. Then I will try to fix this issue.
